api/src/models/master/permitAmendment.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = require('../../utils/model-schema-generator')(
+  'PermitAmendment',
+  {
+    _schemaName: { type: String, default: 'PermitAmendment', index: true },
+    _sourceRefId:  { type: String, default: null, index: true },
+
+    read: [{ type: String, trim: true, default: 'sysadmin' }],
+    write: [{ type: String, trim: true, default: 'sysadmin' }],
+
+    _flavourRecords: [{ type: 'ObjectId', default: [], index: true }],
+
+    statusCode: { type: String, default: '' },
+    typeCode: { type: String, default: '' },
+    receivedDate: { type: Date, default: null },
+    issueDate: { type: Date, default: null },
+    authorizedEndDate: { type: Date, default: null },
+    description: { type: String, default: '' },
+    documents: [{ type: 'ObjectId', default: [], index: true }],
+
+    dateAdded: { type: Date, default: Date.now() },
+    dateUpdated: { type: Date, default: null },
+    datePublished: { type: Date, default: null },
```

Comment:
Can remove the `datePublished` and `publishedBy` from the master record, as they should never themselves get published.

Reply:
Removed from master.